### PR TITLE
Windows compatibility

### PR DESCRIPTION
**Pull request by gregory-latinier**

### CODE REVIEW

File: src/app/Sidebar.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ export default class Sidebar extends Component {
     const { auth, app, hideSidebar } = this.props;
     const { user } = auth;
     const { rate, sidebarLoading, props = {}, categories } = app;
-
+    console.log(formatter);
```
Comment (bonustrack) [inline src/app/Sidebar.js]: Can you remove this?

**Comment by bonustrack** [comment]
The yarn.lock file should be udpated

**Comment by bonustrack** [comment]
Working fine for me in localhost but Heroku deployment fail with this error `sh: 1: gulp: not found`.

File: .eslintrc.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ module.exports = {
   },
   "extends": "airbnb",
   "rules": {
+    "linebreak-style": 0,
```
Comment (Sekhmet) [inline .eslintrc.js]: Can we delete this line? We don't want to have multiple different line endings in our code. Just force users to use LF. They can configure their environment to use those.

Comment (gregory-latinier) [inline .eslintrc.js]: Yes you're right.
I'll change it on all project and configure my IDE accordingly

**Comment by bonustrack** [comment]
The deployment is now failing on Heroku because missing the dependency "del". `Error: Cannot find module 'del'`

**Comment by bonustrack** [comment]
Working 👍

**Pull request merged**